src/components/NcAppSettingsSection/NcAppSettingsSection.vue

```diff
@@ -66,7 +66,7 @@ export default {
 .app-settings-section {
 	margin-bottom: 80px;
 	&__name {
-		font-size: 20px;
+		font-size: 26px;
```

Comment:
```suggestion
		font-size: 1.6em;
```

Sorry :see_no_evil: 
https://github.com/nextcloud/server/pull/46587